app/game-master/action-creators/score.js

```diff
@@ -0,0 +1,10 @@
+import { createAction } from 'redux-actions';
+import {
+    PARTICIPANT_JOINED,
+    PARTICIPANT_LEFT,
+    PARTICIPANT_SOLUTION
+} from '../actions/score';
```

Comment:
As for me: `PARTICIPANT_JOINED` and `PARTICIPANT_LEFT` are `participant.js` actions. `PARTICIPANT_SOLUTION` - `round.js`. Of course - discussable.